clinica/utils/atlas.py

```diff
@@ -355,20 +338,9 @@ def get_name_atlas(): return "Hammers"
     @staticmethod
     def get_atlas_labels():
         import os
-        import platform
```

Comment:
@arnaudmarcoux : I converted these lines into `check_spm()` function. If I refactored correctly, this corresponds to what you described in this [link](https://github.com/aramis-lab/clinica/pull/36#discussion_r328174774).

(BTW, I added a task regarding the handling of SPM-Standalone)